src/System.Security.AccessControl/src/Interop/Interop.Manual.cs

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security.Principal;
+using Microsoft.Win32.SafeHandles;
+
+internal partial class Interop
```

Comment:
Please update the interop code to follow the guidelines outlined at https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md